spec/components/stories/footer_referentiel_component_stories.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class FooterReferentielComponentStories < ViewComponent::Storybook::Stories
+  story :default do
+    constructor
+  end
```

Comment:
```suggestion
  story :default do
      type = select(%i[generique cafe-de-la-place], :generique)
    constructor(type)
  end
```

Il faut lui passer un type sinon le composant ne peut pas se construire dans storybook. On peut faire comme pour le composant EllipseComponentStories et utiliser un select par exemple.